R/qenv-get_code.R

```diff
@@ -2,7 +2,8 @@
 #'
 #' @name get_code
 #' @param object (`qenv`)
-#' @param deparse (`logical(1)`) if the returned code should be converted to character.
+#' @param parse (`logical(1)`) if the returned code should be converted to expression.
```

Comment:
This package is on CRAN already let's not introduce breaking changes. We can still keep `deparse` but have a inverted logic in the code. 
 
```suggestion
#' @param deparse (`logical(1)`) if the returned code should be a `character` instead of `expression`.
```

Reply:
ah yeah! sorry! duly noted!